Source/Core/DolphinQt/Debugger/MemoryViewWidget.cpp

```diff
@@ -216,7 +233,7 @@ void MemoryViewWidget::SetBPType(BPType type)
   m_bp_type = type;
 }
 
-void MemoryViewWidget::SetAddress(u32 address)
+void MemoryViewWidget::SetAddress(u32 address)  // On change memory address in top box
```

Comment:
Ditto. Even, then, documentation of a class interface belongs in the header, not the cpp file.